common/v2/services/useInterval.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef } from 'react';
```

Comment:
Isn't this more like a helper function and not a service?

Reply:
Indeed!
Moved to `v2/utils`